### PR TITLE
chore(docs): Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by captainsafia**
Addresses some of #894.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -45,4 +45,33 @@ comes in really handy when we need to determine what kinds of information should
...
 Good release messages means more informed users means a better project to use. Yay!
 
+Note that this standard applies to both your commit messages and PR titles so you'll
+need to draft the appropriate commit message when you run `git commit -m` or use the
+interface on your visual git interface.
+
+You can use the following verbs as part of your commit messages/PR titles.
+* fix: For when you have fixed a bug.
+* feat: For when you have added a new future.
+* chore: For when you've done a small chore on the repository such as updating
```
Comment (rgbkrk) [inline CONTRIBUTING.md:55]: chores can also include linting

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -45,4 +45,33 @@ comes in really handy when we need to determine what kinds of information should
 go into our release documentation (as the word changelog in the title might suggest!).
 Good release messages means more informed users means a better project to use. Yay!
 
+Note that this standard applies to both your commit messages and PR titles so you'll
+need to draft the appropriate commit message when you run `git commit -m` or use the
+interface on your visual git interface.
+
+You can use the following verbs as part of your commit messages/PR titles.
+* fix: For when you have fixed a bug.
+* feat: For when you have added a new future.
```
Comment (rgbkrk) [inline CONTRIBUTING.md:54]: The important distinction with these is that `fix` and `feat` are the prefixes that we use to create changelogs (other prefixes are ignored, which means they can be anything).

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -45,4 +45,33 @@ comes in really handy when we need to determine what kinds of information should
...
+* If you've made a change to an epic, use the name of the epic exported, such as
+`executeCellEpic`.
+
+## How fast will my PR be merged?
+
+Your pull request will be merged as soon as there are maintainers to review it and
+after tests have passed. You might have to make some changes before your PR is merged
+but as long as you adhere to the steps above and try your best, you should have no problem
+getting your PR merged.
+
```
Comment (rgbkrk) [inline CONTRIBUTING.md:76]: 👍

**Comment by rgbkrk** [comment]
Excellent.

**Review by rgbkrk — APPROVED** [review]

**Comment by rgbkrk** [comment]
Forgot to note - _we can_ declare whatever prefixes we want to use. Right now we're running under the assumption that we're similar to Angular (given what we link to for the conventional changelog standard). For us, a breaking change isn't in terms of the JS APIs but the userland behavior. However, if/when we export our packages then we will have a strict contract of API behavior.

**Comment by jdetle** [comment]
^ Confession time: I had no idea there were specific verbs that we had to use. Oops.

**Comment by rgbkrk** [comment]
I'm guessing the README needs a once-over as well.

File: CONTRIBUTING.md (hunk 2 above)
Comment (stephymiehle) [reply CONTRIBUTING.md:54]: Whoops, typo here on 54! Unless you guys are wizards or something :)

Comment (captainsafia) [reply CONTRIBUTING.md:54]: Welp! You're totally right, @blindingstars. I'll update in another PR that includes the updates to the README.

**Pull request merged**